packages/logger/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@jwalab/logger",
+  "version": "0.0.1",
+  "description": "",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "devDependencies": {
+    "@tsconfig/node14": "^1.0.1",
+    "@typescript-eslint/eslint-plugin": "^4.4",
+    "@typescript-eslint/parser": "^4.4",
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^8.3.0",
+    "openapi-types": "^9.3.0",
+    "prettier": "^2.3.2",
+    "typescript": "^4.4"
+  },
+  "dependencies": {
+      "node-dependency-injection": "^2.7.1",
+      "winston": "^3.3.3"
+  },
+  "scripts": {
+    "prettier": "npx prettier --write .",
+    "lint": "eslint ./src/ --ext .ts --max-warnings 0",
+    "clean": "rm -rf ./dist/*",
+    "dev": "npx tsc -w",
+    "build": "npx tsc -p tsconfig.pkg.json"
```

Comment:
I don't know if it is meaningful to do it here or if should only be on "applications/services" but the use of the `--sourcemap` option is very handy ! see also https://www.npmjs.com/package/source-map-support

Reply:
yes we should definitely include source maps 👍

Reply:
Already enabled in ```tsconfig.build.json``` :)